js/components/multi-audio/multiAudioPopover.js

```diff
@@ -0,0 +1,62 @@
+var React = require('react'),
+  CONSTANTS = require('../../constants/constants'),
+  AccessibleMenu = require('../higher-order/accessibleMenu'),
+  CloseButton = require('../closeButton');
+
+var MultiAudioPopover = React.createClass({
+  /**
+   * close the popover
+   */
+  handleClose: function() {
+    this.props.togglePopoverAction({
```

Comment:
Do we need a null check here?

Reply:
no

Reply:
Why not? Someone else could use this component later on and not pass this function. This parameter is also not listed as required in the proptypes

Reply:
\+